lib/fastlane/actions/xcodebuild.rb

```diff
@@ -85,9 +92,6 @@ def self.run(params)
             # If not passed, retrieve path from previous xcodebuild call
             params[:archive_path] ||= Actions.lane_context[SharedValues::XCODEBUILD_ARCHIVE]
 
-            # Default to ipa as export format
-            params[:export_format] ||= "ipa"
```

Comment:
I'm sorry. But I don't really understand your question. But ...
- I didn't remove `:export_format` option. I've only removed default value for it.
- If user provides `:export_format` but not `:exportOptionsPlist`, then everything will be working like now. The "old" mode from Xcode 6.* will be used.
- If user provides `:export_format` and `:exportOptionsPlist`, he will get the error described above. This is why I've removed default value of `:export_format`. I want to mention that `-exportFormat` is deprecated option "replaced" by `-exportOptionsPlist`.
- Most user doesn't provide `:export_format` in the Fastlane file and in this case everything will be working, because `-exportFormat` is an optional parameter and can be safely omitted. I've tested it on the Xcode 6.4 and Xcode 7.2 for iPhone project.